classes/FaviconController.php

```diff
@@ -13,7 +12,7 @@ public function index()
     {
         $settings = Settings::instance();
 
-        if (!$settings->favicon_enabled) {
+        if (!$settings->webmanifest_enabled) {
```

Comment:
Why we're checking for webmanifest in favicon? I don't get something. If it's reasonable, please explain that in comment right above the conditional to make it clear for all developers.